pkg/core/native/notary.go

```diff
@@ -398,6 +417,38 @@ func (n *Notary) setMaxNotValidBeforeDelta(ic *interop.Context, args []stackitem
 	return stackitem.Null{}
 }
 
+// getNotaryServiceFeePerKey is Notary contract method and returns a reward per notary request key for notary nodes.
+func (n *Notary) getNotaryServiceFeePerKey(ic *interop.Context, _ []stackitem.Item) stackitem.Item {
+	return stackitem.NewBigInteger(big.NewInt(int64(n.GetNotaryServiceFeePerKey(ic.DAO))))
+}
+
+// GetNotaryServiceFeePerKey is an internal representation of Notary getNotaryServiceFeePerKey method.
+func (n *Notary) GetNotaryServiceFeePerKey(dao *dao.Simple) int64 {
+	n.lock.RLock()
+	defer n.lock.RUnlock()
+	if n.isValid {
+		return n.notaryServiceFeePerKey
+	}
+	return getIntWithKey(n.ID, dao, notaryServiceFeeKey)
+}
+
+// setNotaryServiceFeePerKey is Notary contract method and sets a reward per notary request key for notary nodes.
+func (n *Notary) setNotaryServiceFeePerKey(ic *interop.Context, args []stackitem.Item) stackitem.Item {
+	value := toInt64(args[0])
+	// TODO: forbid zero value and add maximum check?
```

Comment:
Not that important for now. Some maximum can be suggested in C# PR.

Reply:
https://github.com/neo-project/neo/pull/2661#discussion_r816894937